paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -631,13 +631,16 @@ def get_slack_client(self) -> SlackClient:
         return get_slack_client().sc
 
     def get_slack_channel(self) -> str:
-        """ Safely get some slack channel to post to. Defaults to ``#test``.
+        """ Safely get some slack channel to post to. Defaults to ``#deploy``.
         Currently only uses the first slack channel available, and doesn't support
         multi-channel notifications. """
-        try:
-            return self.deploy_info.get('slack_channels', ["#test"])[0]
-        except (IndexError, AttributeError):
-            return "#test"
+        if self.deploy_info.get('slack_notify', True) is True:
```

Comment:
`is True` seems unnecessary. Schemas are a better way to be strict about this being a boolean in deploy.yaml.

Reply:
It is unnecessary, I just thought it read better